corona-warn-app/cwa-app-android
# CheckIns Consent (EXPOSUREAPP-6530)

- Implements CheckIns consent screen

### TODO
### Testing
- Add some completed check-ins  
- Go to testing menu -> presence tracing -> scroll to `Consent` button -> Open the new  consent screen
- Select - unselect items
- Check when activity is killed that state is reserved

Note: wiring the new screen will be done in a new PR
